.github/workflows/fmt.yml

```diff
@@ -1,11 +1,24 @@
-on: [push, pull_request, pull_request_target]
+on: push
 
 jobs:
   fmt:
     runs-on: ubuntu-latest
     steps:
+      - name: Configure git user
+        run: |
+          if [[ -n "${{ secrets.PANTHER_BOT_AUTOMATION_TOKEN }}" ]]; then
+            echo "GIT_USER_NAME=panther-bot-automation" >> $GITHUB_ENV
+            echo "GIT_USER_EMAIL=github-service-account-automation@panther.io" >> $GITHUB_ENV
+            echo "GIT_TOKEN=${{ secrets.PANTHER_BOT_AUTOMATION_TOKEN }}" >> $GITHUB_ENV
+          else
+            echo "GIT_USER_NAME=${{ github.actor }}" >> $GITHUB_ENV
+            echo "GIT_USER_EMAIL=$(git log -n 1 --pretty=format:%ae)" >> $GITHUB_ENV
+            echo "GIT_TOKEN=${{ github.token }}" >> $GITHUB_ENV
```

Comment:
We will fall back to the default token in fork PRs, and because of this the workflow will not be re-triggered